pkg/apis/garden/validation/validation_test.go

```diff
@@ -4778,6 +4778,37 @@ var _ = Describe("validation", func() {
 				}))
 			})
 
+			Context("Zoned", func() {
+				var (
+					specField = "spec.cloud.azure.zones"
+					azZones   = []string{"1", "2"}
+				)
+				It("should forbid to move a zoned shoot into a non zoned shoot", func() {
+					shoot.Spec.Cloud.Azure.Zones = azZones
+					newShoot := prepareShootForUpdate(shoot)
+					newShoot.Spec.Cloud.Azure.Zones = []string{}
+
+					errorList := ValidateShootUpdate(newShoot, shoot)
+					Expect(errorList).To(HaveLen(1))
+					Expect(*errorList[0]).To(MatchFields(IgnoreExtras, Fields{
```

Comment:
Can you use `ConsistsOf` instead, please? Then you also don't need to check for the length of the list.

Reply:
done